Makefile

```diff
@@ -451,3 +451,8 @@ package-rpm-suse package-windows package-standalone-macosx \
 package-generic-unix: $(PACKAGES_SOURCE_DIST_FILE)
 	$(verbose) $(MAKE) -C packaging $@ \
 		SOURCE_DIST_FILE=$(abspath $(PACKAGES_SOURCE_DIST_FILE))
+
+generate-config:
+	rm -rf generated/rabbitmq.*.config
+	./cuttlefish -e . -s ./schema -c rabbitmq.conf.d/rabbitmq.conf -f rabbitmq
```

Comment:
There won't be but that's not my point. We use `erlang.mk` to manage dependencies. Bundling a binary goes against that. Using `erlang.mk` to manage this dependency and having `cuttlefish` the binary available at runtime are not mutually exclusive.